packages/swarm-docs/gatsby-config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   siteMetadata: {
-    title: 'Swarm Docs',
+    title: 'Swarm Design System',
```

Comment:
Would it be worth putting 'Meetup' in the title somewhere? 'Swarm Design System - Meetup' or 'Meetup - Swarm Design System'? Mostly thinking about SEO and 'marketing' reasons if this is publicly accessible

Reply:
That makes sense, I'm going to make it `Meetup Swarm Design System Documentation` for SEO purposes.